packages/framework/src/Migrations/Version20191112122617.php

```diff
@@ -0,0 +1,181 @@
+<?php
+
+declare(strict_types=1);
+
+namespace Shopsys\FrameworkBundle\Migrations;
+
+use Doctrine\DBAL\Schema\Schema;
+use Shopsys\MigrationBundle\Component\Doctrine\Migrations\AbstractMigration;
+
+class Version20191112122617 extends AbstractMigration
+{
+    use MultidomainMigrationTrait;
+
+    /**
+     * @param \Doctrine\DBAL\Schema\Schema $schema
+     */
+    public function up(Schema $schema): void
+    {
+        $this->sql('ALTER TABLE payment_domains ADD vat_id INT NOT NULL DEFAULT 1');
+        $this->sql('ALTER TABLE payment_domains ADD CONSTRAINT FK_9532B177B5B63A6B FOREIGN KEY (vat_id) REFERENCES vats (id) NOT DEFERRABLE INITIALLY IMMEDIATE');
+        $this->sql('CREATE INDEX IDX_9532B177B5B63A6B ON payment_domains (vat_id)');
+        $this->sql('ALTER TABLE payment_domains ALTER vat_id DROP DEFAULT');
+
+        $this->sql('ALTER TABLE payment_prices DROP CONSTRAINT payment_prices_pkey');
+        $this->sql('ALTER TABLE payment_prices ADD domain_id INT NOT NULL DEFAULT 1');
+        $this->sql('ALTER TABLE payment_prices ALTER domain_id DROP DEFAULT');
+
+        $this->sql('ALTER TABLE transport_domains ADD vat_id INT NOT NULL DEFAULT 1');
+        $this->sql('ALTER TABLE transport_domains ADD CONSTRAINT FK_18AC7F6CB5B63A6B FOREIGN KEY (vat_id) REFERENCES vats (id) NOT DEFERRABLE INITIALLY IMMEDIATE');
+        $this->sql('ALTER TABLE transport_domains ALTER vat_id DROP DEFAULT');
+
+        $this->sql('ALTER TABLE transport_prices DROP CONSTRAINT transport_prices_pkey');
+        $this->sql('ALTER TABLE transport_prices ADD domain_id INT NOT NULL DEFAULT 1');
+        $this->sql('ALTER TABLE transport_prices ALTER domain_id DROP DEFAULT');
+
+        $this->migratePaymentsDomains();
+        $this->migratePaymentsPrices();
+        $this->migrateTransportDomains();
+        $this->migrateTransportPrices();
+
+        $this->sql('ALTER TABLE payment_prices ADD PRIMARY KEY (payment_id, domain_id)');
```

Comment:
I guess, it is because of my testing, I manually added `currency_id` with value 1, because `INSERT INTO payment_prices (payment_id, price, domain_id...)` was not working, it was expecting `currency_id` (don't know why, need more time to understand the code).
So after my modification, there probably exist 2 rows with the same value of  `payment_id` and `domain_id`.
It would be better to get in touch and consult it. 

After we solve this, I will be able to continue with testing

Reply:
Hello, thx for this comment, I fixed it.